Stride.BepuPhysics/Components/Containers/BodyContainerComponent.cs

```diff
@@ -41,15 +39,16 @@ public byte MinimumTimestepCountUnderThreshold
             set
             {
                 _minimumTimestepCountUnderThreshold = value;
-                if (ContainerData?.Exist == true)
-                    ContainerData.BuildOrUpdateContainer();
+                ContainerData?.TryUpdateContainer();
             }
         }
 
+        #warning Users should not have to interact with this method directly, have a look at the car component to see how awkward it is to use
+        // the struct doesn't seem safe to store either, what happens if the body is removed from the sim but users still interact with the struct, they're affecting the body that replaced it in that slot, right ?
+        // We could copy the method that struct contains into this component and call them from here, hiding away the additional nonsense we have to deal with
```

Comment:
@Nicogo1705 would be able to speak on this more but I think there was an issue with the body reference and not being able to store the reference for long term use, this was a more reliable way IIRC.

Reply:
Right, what I meant to say there is that we should not provide our users with any `BodyReference` at all, as you said, it's not that safe of a type to manipulate. We should instead think of `BodyContainerComponent` as a wrapper around `BodyReference`, so creating functions and properties in there that would be forwarded to the `BodyReference`.

Reply:
https://github.com/Nicogo1705/Stride.BepuPhysics/issues/20 

(Just for tracking; sorry, i made theses Issues when in a car when i had 15minutes of free times, i will rewrite them)